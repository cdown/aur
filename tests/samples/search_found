{"version":1,"type":"search","resultcount":3,"results":[{"ID":189701,"Name":"poco","PackageBaseID":13487,"PackageBase":"poco","Version":"1.6.0-1","Description":"C++ class libraries for network-centric, portable applications, complete edition with debug libraries","URL":"http:\/\/www.pocoproject.org\/","NumVotes":87,"OutOfDate":0,"Maintainer":"ido","FirstSubmitted":1193569357,"LastModified":1433875194,"License":"custom:boost","URLPath":"\/cgit\/aur.git\/snapshot\/poco.tar.gz","CategoryID":1,"Popularity":0},{"ID":202080,"Name":"flopoco","PackageBaseID":72798,"PackageBase":"flopoco","Version":"2.5.0-1","Description":"A generator of arithmetic cores (Floating-Point Cores, but not only) for FPGAs (but not only)","URL":"http:\/\/flopoco.gforge.inria.fr\/","NumVotes":1,"OutOfDate":0,"Maintainer":"marzoul","FirstSubmitted":1377118989,"LastModified":1435126835,"License":"custom","URLPath":"\/cgit\/aur.git\/snapshot\/flopoco.tar.gz","CategoryID":1,"Popularity":0},{"ID":219477,"Name":"libpoco-basic","PackageBaseID":74539,"PackageBase":"libpoco-basic","Version":"1.6.1-1","Description":"C++ class libraries for network-centric, portable applications, basic edition","URL":"http:\/\/www.pocoproject.org","NumVotes":1,"OutOfDate":0,"Maintainer":"Raimar","FirstSubmitted":1382130102,"LastModified":1439120781,"License":"custom:boost","URLPath":"\/cgit\/aur.git\/snapshot\/libpoco-basic.tar.gz","CategoryID":1,"Popularity":0}]}