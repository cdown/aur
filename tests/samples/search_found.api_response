{"version":5,"type":"search","resultcount":5,"results":[{"ID":284071,"Name":"poco-git","PackageBaseID":108665,"PackageBase":"poco-git","Version":"1.7.0.r0.gf28591b-1","Description":"C++ class libraries for network-centric, portable applications, complete edition","URL":"http:\/\/www.pocoproject.org\/","NumVotes":3,"Popularity":0,"OutOfDate":null,"Maintainer":"andres.manz","FirstSubmitted":1457703284,"LastModified":1457749197,"URLPath":"\/cgit\/aur.git\/snapshot\/poco-git.tar.gz"},{"ID":471841,"Name":"ros-ardent-poco-vendor","PackageBaseID":128251,"PackageBase":"ros-ardent-poco-vendor","Version":"1.0.0-1","Description":"ROS - CMake shim over the poco library.","URL":"http:\/\/www.ros.org\/","NumVotes":0,"Popularity":0,"OutOfDate":null,"Maintainer":"racko","FirstSubmitted":1513897192,"LastModified":1513897193,"URLPath":"\/cgit\/aur.git\/snapshot\/ros-ardent-poco-vendor.tar.gz"},{"ID":648985,"Name":"mingw-w64-poco","PackageBaseID":131346,"PackageBase":"mingw-w64-poco","Version":"1.9.3-1","Description":"C++ class libraries for network-centric, portable applications, complete edition with debug libraries (mingw-w64)","URL":"http:\/\/www.pocoproject.org\/","NumVotes":1,"Popularity":0,"OutOfDate":null,"Maintainer":"Forty-Bot","FirstSubmitted":1522824936,"LastModified":1569167622,"URLPath":"\/cgit\/aur.git\/snapshot\/mingw-w64-poco.tar.gz"},{"ID":764256,"Name":"flopoco","PackageBaseID":72798,"PackageBase":"flopoco","Version":"4.1.2-1","Description":"A generator of arithmetic cores (Floating-Point Cores, but not only) for FPGAs (but not only)","URL":"http:\/\/flopoco.gforge.inria.fr\/","NumVotes":1,"Popularity":0,"OutOfDate":null,"Maintainer":"marzoul","FirstSubmitted":1377118989,"LastModified":1594591525,"URLPath":"\/cgit\/aur.git\/snapshot\/flopoco.tar.gz"},{"ID":793179,"Name":"libpoco-basic","PackageBaseID":74539,"PackageBase":"libpoco-basic","Version":"1.10.1-1","Description":"C++ class libraries for network-centric, portable applications, basic edition","URL":"http:\/\/www.pocoproject.org","NumVotes":1,"Popularity":0,"OutOfDate":null,"Maintainer":"Raimar","FirstSubmitted":1382130102,"LastModified":1600375957,"URLPath":"\/cgit\/aur.git\/snapshot\/libpoco-basic.tar.gz"}]}