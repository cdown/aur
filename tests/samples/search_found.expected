[
    {
        "category_id": 1,
        "description": "C++ class libraries for network-centric, portable applications, complete edition with debug libraries",
        "first_submitted": 1193569357,
        "id": 189701,
        "last_modified": 1433875194,
        "license": "custom:boost",
        "maintainer": "ido",
        "name": "poco",
        "num_votes": 87,
        "out_of_date": false,
        "package_base": "poco",
        "package_base_id": 13487,
        "popularity": 0,
        "url": "http://www.pocoproject.org/",
        "url_path": "/cgit/aur.git/snapshot/poco.tar.gz",
        "version": "1.6.0-1"
    },
    {
        "category_id": 1,
        "description": "A generator of arithmetic cores (Floating-Point Cores, but not only) for FPGAs (but not only)",
        "first_submitted": 1377118989,
        "id": 202080,
        "last_modified": 1435126835,
        "license": "custom",
        "maintainer": "marzoul",
        "name": "flopoco",
        "num_votes": 1,
        "out_of_date": false,
        "package_base": "flopoco",
        "package_base_id": 72798,
        "popularity": 0,
        "url": "http://flopoco.gforge.inria.fr/",
        "url_path": "/cgit/aur.git/snapshot/flopoco.tar.gz",
        "version": "2.5.0-1"
    },
    {
        "category_id": 1,
        "description": "C++ class libraries for network-centric, portable applications, basic edition",
        "first_submitted": 1382130102,
        "id": 219477,
        "last_modified": 1439120781,
        "license": "custom:boost",
        "maintainer": "Raimar",
        "name": "libpoco-basic",
        "num_votes": 1,
        "out_of_date": false,
        "package_base": "libpoco-basic",
        "package_base_id": 74539,
        "popularity": 0,
        "url": "http://www.pocoproject.org",
        "url_path": "/cgit/aur.git/snapshot/libpoco-basic.tar.gz",
        "version": "1.6.1-1"
    }
]